### PR TITLE
Bump SDK version

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

File: sdk.conf [hunk 1]
```diff
@@ -1 +1 @@
-osgeo4a_version=20191211
+osgeo4a_version=20191213
```
Comment (m-kuhn) [inline sdk.conf]:
```suggestion
osgeo4a_version=20191216
```

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-742-475b07176c8f205e14e336ee38f8c2413c9e87bb-armv7.apk)
 - [arm64_v8a](https://download.opengis.ch/qfield/ci-builds/qfield-dev-742-475b07176c8f205e14e336ee38f8c2413c9e87bb-arm64_v8a.apk)
 - [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-742-475b07176c8f205e14e336ee38f8c2413c9e87bb-x86.apk)
 - [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-742-475b07176c8f205e14e336ee38f8c2413c9e87bb-x86_64.apk)

**Comment by signedav** [comment]
Looks good @m-kuhn

**Comment by m-kuhn** [comment]
Cool, all PRs are merged on master, let's prepare a new sdk and rebuild

File: .travis.yml [hunk 2]
```diff
@@ -31,7 +29,7 @@ script:
 
 jobs:
   allow_failures:
-#    - name: "🍳 Testing"
+    - name: "🍳 Testing"
```
Comment (m-kuhn) [inline .travis.yml]:
```suggestion
#    - name: "🍳 Testing"
```

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-742-b4bc46120e50f44a411fcda2a23501260440caa9-armv7.apk)
 - [arm64_v8a](https://download.opengis.ch/qfield/ci-builds/qfield-dev-742-b4bc46120e50f44a411fcda2a23501260440caa9-arm64_v8a.apk)
 - [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-742-b4bc46120e50f44a411fcda2a23501260440caa9-x86.apk)
 - [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-742-b4bc46120e50f44a411fcda2a23501260440caa9-x86_64.apk)

**Pull request merged**